cdc/owner.go

```diff
@@ -34,6 +35,8 @@ import (
 	"go.uber.org/zap"
 )
 
+var markProcessorDownTime = 5 * time.Minute
```

Comment:
@amyangfei just increase to be 5 minute now.
I think the better implementation should check the aliveness of capture, like maintain a session in etcd.

Reply:
Oh I mistake it at first, it is one minute, not one second. Five minutes is too long for CDC owner to detect processor down 🙁️
maybe we can recover it to 1 minute for the test, and have a better detect way in other PR.

Reply:
reverted